src/preprocess/query_generator.py

```diff
@@ -2,23 +2,23 @@
 
 
 def remove_all_unnecessary_dependencies(f):
-    f.write("""delete from programstageinstance_messageconversation;\n""")
-    f.write("""delete from programinstancecomments;\n""")
-    f.write("""delete from programinstanceaudit;\n""")
-    f.write("""delete from datavalueaudit;\n""")
-    f.write("""delete from trackedentitydatavalueaudit;\n""")
-    f.write("""delete from trackedentityattributevalueaudit;\n""")
-    f.write("""delete from programstageinstance_messageconversation;\n""")
-    f.write("""delete from dataapprovalaudit;\n""")
-    f.write("""delete from interpretation_comments;\n""")
-    f.write("""delete from interpretationcomment;\n""")
-    f.write("""delete from interpretationusergroupaccesses;\n""")
-    f.write("""delete from intepretation_likedby;\n""")
-    f.write("""delete from messageconversation_messages;\n""")
-    f.write("""delete from messageconversation_usermessages;\n""")
-    f.write("""delete from messageconversation;\n""")
-    f.write("drop view if exists _view_nhwa_data_audit;")
-    f.write("drop view if exists _view_test2;\n")
+    f.write("""delete from programstageinstance_messageconversation;\n
+    delete from programinstancecomments;\n
+    delete from programinstanceaudit;\n
+    delete from datavalueaudit;\n
+    delete from trackedentitydatavalueaudit;\n
+    delete from trackedentityattributevalueaudit;\n
+    delete from programstageinstance_messageconversation;\n
+    delete from dataapprovalaudit;\n
+    delete from interpretation_comments;\n
+    delete from interpretationcomment;\n
+    delete from interpretationusergroupaccesses;\n
+    delete from intepretation_likedby;\n
+    delete from messageconversation_messages;\n
+    delete from messageconversation_usermessages;\n
+    delete from messageconversation;\n
+    drop view if exists _view_nhwa_data_audit;
+    drop view if exists _view_test2;\n""")
```

Comment:
`\n` can be removed on a multi-line string. The good thing about this way is that you can simply paste a normal SQL